app/views/measurements/_form.html.erb

```diff
@@ -11,6 +11,29 @@
     </div>
   <% end %>
 
+  <% if request.path =~ /edit/ %>
```

Comment:
Putting this component in the edit view would remove the need for the conditional in the template.

Reply:
Excellent idea.